app/src/main/java/org/mozilla/fenix/compose/CollectionsPlaceholder.kt

```diff
@@ -0,0 +1,167 @@
+ *
+ * @param showAddToCollectionButton Whether or not the "Add to Collection" button should be shown.
+ * @param onAddTabsToCollectionButtonClick Invoked when the user clicks on the "Add Tabs to Collection" button.
+ * @param onRemovePlaceholderClick Invoked when the user clicks on the close button to remove the Collections
+ * placeholder.
+ */
+@Composable
+fun CollectionsPlaceholder(
+    showAddToCollectionButton: Boolean,
+    onAddTabsToCollectionButtonClick: () -> Unit,
+    onRemovePlaceholderClick: () -> Unit,
+) {
+    Box(
+        modifier = Modifier
+            .semantics(mergeDescendants = true) {}
+            .dashedBorder(
+                color = FirefoxTheme.colors.borderPrimary,
+                cornerRadius = 8.dp,
+                dashHeight = 2.dp,
+                dashWidth = 4.dp
+            )
+    ) {
+        Column(
+            Modifier
+                .padding(
+                    start = 16.dp,
+                    end = 16.dp,
+                    bottom = 16.dp
+                )
+                .fillMaxWidth()
+        ) {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+                SectionHeader(
+                    text = stringResource(R.string.collections_header),
+                    modifier = Modifier
+                        .weight(1f)
+                        .wrapContentHeight(align = Alignment.Top)
+                )
+
+                IconButton(
+                    onClick = onRemovePlaceholderClick
+                ) {
+                    Icon(
+                        painter = painterResource(R.drawable.ic_close),
+                        contentDescription = stringResource(
+                            R.string.remove_home_collection_placeholder_content_description
+                        ),
+                        modifier = Modifier.size(24.dp),
+                        tint = FirefoxTheme.colors.iconPrimary
+                    )
+                }
+            }
+
+            SecondaryText(
+                text = stringResource(R.string.no_collections_description2),
+                modifier = Modifier.fillMaxWidth(),
+                fontSize = 14.sp
+            )
+
+            if (showAddToCollectionButton) {
+                Spacer(modifier = Modifier.height(8.dp))
+
+                AddToCollectionButton(
+                    onClick = onAddTabsToCollectionButtonClick
+                )
+            }
+        }
+    }
+}
+
+/**
+ * Add to Collections button.
+ *
+ * @param onClick Invoked when the user clicks on the button.
+ */
+@Composable
+fun AddToCollectionButton(onClick: () -> Unit) {
+    Button(
+        onClick = onClick,
+        modifier = Modifier
+            .clip(RoundedCornerShape(size = 4.dp))
+            .fillMaxWidth()
+            .height(35.dp),
```

Comment:
Wondering where you got this 35.dp? Shouldn't this be 48.dp? https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/values/styles.xml#346